klass-api/pom.xml

```diff
@@ -198,6 +198,9 @@
                                 <include>**/*Test.java</include>
                                 <include>**/*Documentation.java</include>
                             </includes>
+                            <excludes>
+                                <exclude>/migration/**</exclude>
```

Comment:
Not sure that this path is correct? Normally the leading `/` refers to the root of the repo